sdk/core/core-rest-pipeline/src/createPipelineFromOptions.ts

```diff
@@ -69,12 +65,10 @@ export function createPipelineFromOptions(options: InternalPipelineOptions): Pip
   }
 
   pipeline.addPolicy(formDataPolicy());
-  pipeline.addPolicy(tracingPolicy(options.userAgentOptions));
   pipeline.addPolicy(userAgentPolicy(options.userAgentOptions));
   pipeline.addPolicy(setClientRequestIdPolicy());
-  pipeline.addPolicy(throttlingRetryPolicy(), { phase: "Retry" });
-  pipeline.addPolicy(systemErrorRetryPolicy(options.retryOptions), { phase: "Retry" });
-  pipeline.addPolicy(exponentialRetryPolicy(options.retryOptions), { phase: "Retry" });
+  pipeline.addPolicy(defaultRetryPolicy(), { phase: "Retry" });
+  pipeline.addPolicy(tracingPolicy(options.userAgentOptions), { phase: "Retry" });
```

Comment:
This change is to make retries properly reflected on the tracing spans 🙂

Reply:
How about `afterPhase: "Retry"` - would that work here? Then it won't break when someone shuffles the order of calls to `addPolicy` assuming I understand how it works 😄

Reply:
And just to confirm - you still get 1 span when there are no retries right?

Reply:
It should be the case, but I’ll confirm!

Reply:
`afterPhrase: “Retry”` is the way to go! Thank you